src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -202,7 +202,10 @@ def get_labels(df_row: pd.Series[Any]) -> LanguageDict:
     Returns:
         A dictionary with the language tag and the content of the cell
     """
-    return LanguageDict({lang: df_row[f"label_{lang}"] for lang in languages if not pd.isna(df_row[f"label_{lang}"])})
+    labels = {lang: labl for lang in languages if not pd.isna(labl := df_row[f"label_{lang}"])}
```

Comment:
```suggestion
    labels = {lang: label for lang in languages if not pd.isna(label := df_row[f"label_{lang}"])}
```

Reply:
I'm fascinated that `pd.isna(labl := df_row[f"label_{lang}"])` even works. So cool!

Reply:
yes! It does for a lot. If you turn something into a bool, then it is important that the extraction happens inside brackets so that the whole expression is evaluated but the variable does not contain the boolean expression.